File: src/FinTrack.Application/DTOs/Authentication/AuthenticationResponse.cs
namespace FinTrack.Application.DTOs.Authentication
{
    public class AuthenticationResponse
    {
        public required string AccessToken { get; set; }
        public required string RefreshToken { get; set; }
        public DateTime ExpiresAt { get; set; }

        public required UserSummaryResponse User { get; set; }


    }
}

File: src/FinTrack.Client/src/app/core/models/auth/authentication-response.ts
export interface AuthenticationResponse
{
    accessToken:string;
    refreshToken:string;
    user:UserSummaryResponse;
}

export interface UserSummaryResponse
{
    id:string;
    firstName:string;
    lastName:string;
    email:string;
    currency:string;
    timeZone:string;
}